### PR TITLE
Mirgrating unique Partition chart controls

### PR DESCRIPTION
**Pull request by rusackas**
### CATEGORY

Choose one

- [ ] Bug Fix
- [ ] Enhancement (new features, refinement)
- [x] Refactor
- [ ] Add tests
- [ ] Build / Development Environment
- [ ] Documentation

### SUMMARY

Moving unique control(s) from `controls.jsx` to `/explore/controlPanels/Partition.jsx`

### ADDITIONAL INFORMATION

- [x] Has associated issue: #9187
- [ ] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### REVIEWERS
@kristw

### CODE REVIEW

**Comment by rusackas** [comment]
@kristw note that this one is a little peculiar. It uses some JSX, importing and passing a component. We'll probably need a way to displace that dependency to move it to the plugins repo.

**Review by mistercrunch — APPROVED** [review]

**Review by villebro — APPROVED** [review]
This is one of the more exotic post processing visualizations that might benefit from moving the post processing to the frontend. LGTM, but let's tag this as one to refactor down the road.

File: superset-frontend/src/explore/controlPanels/Partition.jsx [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+import { t } from '@superset-ui/translation';
+import { NVD3TimeSeries } from './sections';
+import OptionDescription from '../../components/OptionDescription';
+import * as v from '../validators';
```
Comment (kristw) [inline superset-frontend/src/explore/controlPanels/Partition.jsx]: Could we also refactor to do specific imports?

```ts
import { nonEmpty } from '../validators';
```

**Review by kristw — APPROVED** [review]

File: superset-frontend/src/explore/controlPanels/Partition.jsx (hunk 1 above)
Comment (rusackas) [reply superset-frontend/src/explore/controlPanels/Partition.jsx]: Absolutely.

**Pull request merged**